
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


# conda environments:
#
summit                *  /home/kcmf2/.conda/envs/summit
testenv                  /home/kcmf2/.conda/envs/testenv
base                     /usr/local/software/spack/spack-0.11.2/opt/spack/linux-rhel7-x86_64/gcc-5.4.0/miniconda3-4.5.4-hivczbzklvoccmuifapprxz7humnmn5c
bioenv                   /usr/local/software/spack/spack-0.11.2/opt/spack/linux-rhel7-x86_64/gcc-5.4.0/miniconda3-4.5.4-hivczbzklvoccmuifapprxz7humnmn5c/envs/bioenv
openmm-env               /usr/local/software/spack/spack-0.11.2/opt/spack/linux-rhel7-x86_64/gcc-5.4.0/miniconda3-4.5.4-hivczbzklvoccmuifapprxz7humnmn5c/envs/openmm-env

Changed directory to /home/kcmf2/summit_private/experiments/tsemo.

JobID: 25293387
======
Time: Sat Jun 27 22:09:47 BST 2020
Running on master node: cpu-e-270
Current directory: /home/kcmf2/summit_private/experiments/tsemo

Nodes allocated:
================
cpu-e-270

numtasks=1, numnodes=1, mpi_tasks_per_node=1 (OMP_NUM_THREADS=1)

Executing command:
==================
python tsemo_dtlz2_test.py 

Traceback (most recent call last):
  File "tsemo_dtlz2_test.py", line 1, in <module>
    from summit.strategies import TSEMO
ModuleNotFoundError: No module named 'summit'
